sql_generators/kpi_support_metrics/templates/active_users.view.sql

```diff
@@ -0,0 +1,53 @@
+{{ header }}
+CREATE OR REPLACE VIEW
+  `{{ project_id }}.{{ dataset }}.{{ name }}`
+AS
+SELECT
+  *,
```

Comment:
Sorry, I didn't catch the `SELECT *` on my first review.

We don't actually _need_ everything from `baseline_clients_last_seen` -- `submission_date`, `client_id`, `sample_id`, `days_seen_bits`, and `days_active_bits` is sufficient. So this view could be more lightweight. But if DE prefers to have all of the columns available for other tasks (e.g. writing `active_users_aggregates`) then keeping the `SELECT *` works for me.